src/DefaultProgressBar.jsx

```diff
@@ -0,0 +1,80 @@
+import PropTypes from 'prop-types';
+import { forbidExtraProps } from 'airbnb-prop-types';
+import React from 'react';
+
+import { withStyles, withStylesPropTypes } from 'react-with-styles';
+
+const propTypes = forbidExtraProps({
+  ...withStylesPropTypes,
+  orientation: PropTypes.string,
+  disabled: PropTypes.bool,
+  style: PropTypes.any,
+});
+
+const defaultProps = {
+  orientation: 'horizontal',
+  disabled: false,
+  style: {},
+};
+
+function DefaultProgressBar({
+  css,
+  styles,
+  theme, // eslint-disable-line no-unused-vars
+  orientation,
+  disabled,
+  ...passProps
+}) {
+  return (
+    <div
```

Comment:
@ljharb that's probably one of the a11y fixes in our pipeline... for when we've consolidate on one rheostat. :P I don't think it makes sense to include that fix in this PR esp as it may have other repercussions.